client/chat/channel.jsx

```diff
@@ -71,6 +71,18 @@ const UserSublist = styled.ul`
   list-style-type: none;
   margin: 0;
   padding: 0;
+
+  ${props => {
+    if (props.sectionTitle === 'Offline') {
+      return `
+        color: ${colorTextFaint};
+        svg {
```

Comment:
This selector is definitely more general than it needs to be and might cause problems in the future. Prefer a more direct component reference, e.g.
```
${StyledAvatar} {
  opacity: ...
}
```

Reply:
Would you prefer the opacity here to be changed to `alphaSecondary` so that it matches the color change to `colorTextSecondary`?

Reply:
I don't have too much of a preference, generally icons can withstand a lower opacity and still be visible, so you just want it to look visually "the same" (which, counterintuitively, often means icons end up with a lower opacity than text). I haven't looked to see what feels like it matches in this particular case, but I feel like the lower opacity here is probably okay.